script/tictactoe.js

```diff
@@ -1,113 +1,97 @@
 $(document).ready(function(){
 
-	let count=0;
-
-	let player1_row_1 = 0;
-	let player1_row_2 = 0;
-	let player1_row_3 = 0;
-	let player1_col_1 = 0;
-	let player1_col_2 = 0;
-	let player1_col_3 = 0;
-
-	let player2_row_1 = 0;
-	let player2_row_2 = 0;
-	let player2_row_3 = 0;
-	let player2_col_1 = 0;
-	let player2_col_2 = 0;
-	let player2_col_3 = 0;
+	var clickCount=0;
+	var result,player,playerToken;
+	
 
-	let player1_diag_1 = 0;
-	let player1_diag_2 = 0;
+	$(".box").click(function(){
+		$(this).unbind('click');
+		clickCount+=1;
+		setPlayerToken(clickCount,$(this));
+		showPlayerTurn();
 
-	let player2_diag_1 = 0;
-	let player2_diag_2 = 0;
-	
-	let draw,finished,winner; 
+		if (clickCount>=5){
+			checkRowWinner();
+			checkColWinner();
+			checkDiagWinner();
+		}
 
-	$(".box").css("pointer-events", "none");
+		if(clickCount === 9 && !result){
+			checkDraw();
+		}
 
-		$(".box").click(function(){
-			if (count <= 9) {
-				count+=1;
-				id = $(this).attr("id")
-				if (count % 2 != 0){
-					//player 1
-					$(this).addClass("disabled1")
-					$(this).html("O")
+	});
 
-					$(".player1").hide()
-					$(".player2").show()
+	//for statrting game
+		$("#start").click(function(){
+			$(".box").css("pointer-events", "auto");
+			$("#start").hide();
+			$(".player1").show();
+		});
 
-					id.split("")[0]== "1"? player1_row_1+=1: id.split("")[0]== "2"?player1_row_2+=1:player1_row_3+=1;
-					id.split("")[1]== "1"? player1_col_1+=1: id.split("")[1]== "2"?player1_col_2+=1:player1_col_3+=1;
-					if (id.split("")[0]== id.split("")[1]){player1_diag_1+=1}
-					if (id == "13" || id == "22" || id == "31"){player1_diag_2+=1}
-					
-				}
-				else{
-					//player 2
-					$(this).addClass("disabled2")
-					$(this).html("X")
-					
-					$(".player1").show()
-					$(".player2").hide()
+	//function for setting the player token
+	function setPlayerToken(clickCount,thisBox){
+		if (clickCount % 2 != 0){
+			thisBox.addClass("disabled1");
+			thisBox.text("O");
+			playerToken="OOO";
+			player="1";
+		}
+		else{
+			thisBox.addClass("disabled2");
+			thisBox.text("X");
+			playerToken="XXX";
+			player="2";
```

Comment:
please create a function for the similar code of the if else condition

Reply:
Ok. Thanks.